apps/angular/signal-input/src/app/user.component.ts

```diff
@@ -29,10 +30,8 @@ const ageToCategory = (age: number): Category => {
 export class UserComponent {
   name = input.required<string>();
   lastName = input('');
-  category = input.required({
-    alias: 'age',
-    transform: (age: string): Category => ageToCategory(Number(age) ?? 0),
-  });
+  age = input.required({ transform: numberAttribute });
 
+  category = computed(() => ageToCategory(this.age()));
```

Comment:
Don't you think this is way easier to read ?

Reply:
You're right, but I was having fun playing with the alias and transform for the first time :)